### DISCUSSION TITLE
Addresses

### DISCUSSION THREAD

**Discussion by Tempofranz** [DeFiCh/app#817] [Ideas]
Please input the Chance of whitelist-accounts!?!

**Comment by angelocastiglione** [comment]
Is a good idea to create address book whitelist. With address book I can send DFI without copy & past the address.

**Comment by angelocastiglione** [comment]
> Please input the Chance of whitelist-accounts!?!

If create address book whitelist i can add my address of cake and send my dfi from app to cake without mistake
